Extension/package.nls.json

```diff
@@ -17,6 +17,64 @@
     "c_cpp.command.rescanWorkspace.title": "Rescan Workspace",
     "c_cpp.command.vcpkgClipboardInstallSuggested.title": "Copy vcpkg install command to clipboard",
     "c_cpp.command.vcpkgOnlineHelpSuggested.title": "Visit the vcpkg help page",
+    "c_cpp.configuration.formatting.engine.description": "Select Formatting Engine",
```

Comment:
I think these should be more descriptive.  i.e. "Specifies which engine is used to perform formatting".  "Indicates whether ...", etc.